test/index.test.js

```diff
@@ -16,15 +16,60 @@
 
```

Comment:
don't you need the `process.env.FORCE_HTTP1` ?

Reply:
Not here.